lib/endoscope/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "endoscope"
+version = "0.1.0"
+edition = "2021"
+readme = "README.md"
+resolver = "2"
+
+#[profile.release]
+#codegen-units = 1
+#debug = 2
+#lto = true
+#opt-level = "z"
+#panic = "abort"
+
+#[profile.dev]
+#codegen-units = 1
+#debug = 2
+#lto = true
+#opt-level = 1
+#panic = "abort"
+
+[dependencies]
+cortex-m = { workspace = true, default-features = false }
+cortex-m-rt = { workspace = true, default-features = false, features = ["device"] }
+panic-halt = { workspace = true, default-features = false }
+sha3 = { workspace = true, default-features = false }
+endoscope-abi = { path = "../endoscope-abi" }
+stm32h7 = { workspace = true }
+drv-stm32h7-startup = { path = "../../drv/stm32h7-startup" }
+
+[build-dependencies]
+anyhow = { workspace = true }
+
+[[bin]]
+name = "endoscope"
+test = false
+doctest = false
+bench = false
+
+[features]
+# running in SRAM with default clocking takes about 6 seconds
+# running in ITCM/DTCM with tuned clocking takes about 0.5 seconds
+soc_stm32h753 = ["stm32h7/stm32h753", "drv-stm32h7-startup/h753"]
+
+#[build]
```

Comment:
What are the benefits to keeping this section (and the ones above) that are commented out?

Reply:
It's a reminder that if lto is ever enabled in bindeps, that it should be used in order to save some time and space.

Reply:
Changing to an explicit reminder.